PostProcessing/Runtime/PostProcessLayer.cs

```diff
@@ -567,6 +580,29 @@ void SetupContext(PostProcessRenderContext context)
             context.antialiasing = antialiasingMode;
             context.temporalAntialiasing = temporalAntialiasing;
             context.logHistogram = m_LogHistogram;
+
+#if UNITY_2018_2_OR_NEWER
+            context.physicalCamera = context.camera.usePhysicalProperties;
+            
+            LookAtConstraint lookAtConstraint = context.camera.gameObject.GetComponent<LookAtConstraint>();
```

Comment:
This will spits garbage on every frame (`GetComponent<LookAtConstraint>()`). It would be better to cache it as this is executed on every frame.